sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -452,311 +506,96 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
-  test("create hive view for joined tables") {
```

Comment:
actually you moved it [here](https://github.com/apache/spark/pull/16674/files#diff-e09027394aebb11c880b375693f59fafR495)